homeworks/dankhazipeter/2_vars_datatypes/fictive_character.py

```diff
@@ -0,0 +1,22 @@
+# Adatok bekérése
+name = input("Add meg a karakter nevét: ")
+age_in_years = input("Add meg a karakter életkorát (évben): ")
+python_experience = input("Add meg a Python tapasztalatot években: ")
+wants_to_be_dev = input(
+    "Szeretné, hogy a karakter profi Python fejlesztő legyen? (yes/no): ")
+
+# Adattisztítás és típuskonverzió
+name = name.strip().title()
+age_in_years = int(age_in_years)
+python_experience = int(python_experience)
+
+# Ternary operator használata a válasz alapján
+wanna_be_pro_dev = True if wants_to_be_dev.lower() == "yes" else False
+
+# Életkor kiszámítása napokban
+# Feltételezzük, hogy ma van a születésnapja, nincs szökőév figyelembe véve
+age_in_days = age_in_years * 365
+
+# Kiírás
+print(f"My character is {age_in_days} days old. His/her name is {name} and he/she has {python_experience} years of experience. "
```

Comment:
Mi lenne, ha a 14. soron a ternary kifejezés csak annyit adna vissza hogy "wants" és "does not want", és a mondat többi részét ("He/she {wanna_be_pro_dev} to be a Python developer!" beraknád a print-be? így nincs ismétlés a printen belül és nem kell még egy ternary sem oda.

Reply:
ok, persze, mindjárt átírom. Én bevallom csak a printf részbe tettem volna a ternaryt, de akkor nem lett volna boolean átalakítás, amit a feladat kért, ezért csináltam így :) Mindjárt küldöm az új változatot, írok, ha kész.

Reply:
bent is van

Reply:
igen, ebben én vagyok a ludas, megfogalmazhattam volna jobban is a feladatot. Köszi szépen!